packages/react/src/SubdomainNavBar/SubdomainNavBar.tsx

```diff
@@ -79,10 +80,22 @@ function Root({
 }: SubdomainNavBarProps) {
   const [menuHidden, setMenuHidden] = useState(true)
   const [searchVisible, setSearchVisible] = useState(false)
+  const {width} = useWindowSize()
 
   const handleMobileMenuClick = () => setMenuHidden(!menuHidden)
   const handleSearchVisibility = () => setSearchVisible(!searchVisible)
 
+  useEffect(() => {
+    if (width && width >= 768) {
```

Comment:
Can you use `isMedium` from `useWindowSize` here to avoid hardcoding this breakpoint?